__tests__/components/AnonymousChat/child/ChatItemMyTextV2.test.tsx

```diff
@@ -1,43 +1,178 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+/* eslint-disable react/display-name */
 import * as React from 'react';
 import FastImage from 'react-native-fast-image';
-import {render} from '@testing-library/react-native';
+import {act, fireEvent, render} from '@testing-library/react-native';
 
 import ChatItemMyTextV2 from '../../../../src/components/AnonymousChat/child/ChatItemMyTextV2';
+import {Context} from '../../../../src/context';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../../src/utils/constants';
 
-describe('TESTING ChatItemTargetText', () => {
+jest.spyOn(React, 'useRef').mockReturnValue({current: {close: jest.fn()}});
+jest.mock('react-native-reanimated', () => ({
+  ...jest.requireActual('react-native-reanimated'),
+  useSharedValue: jest.fn(() => ({value: 0})),
+  useAnimatedStyle: jest.fn(() => ({})),
+  View: require('react-native/Libraries/Components/View/View').View,
+  Animated: {
+    View: require('react-native/Libraries/Components/View/View').View
+  }
+}));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Mocking `react-native-reanimated` and `react-native-gesture-handler` libraries is a good practice for testing, but it might be better to move these mocks into a separate setup file or a mock file to keep your test files clean and focused on tests only.



<!-- This is an auto-generated comment by OSS CodeRabbit -->